course-mw/sunbird-util/sunbird-cassandra-utils/src/main/java/org/sunbird/cassandraimpl/CassandraDACImpl.java

```diff
@@ -32,7 +32,7 @@ public Response getRecords(
     try {
       Select select;
       if (CollectionUtils.isNotEmpty(fields)) {
-        select = QueryBuilder.select((String[]) fields.toArray()).from(keySpace, table);
+        select = QueryBuilder.select(fields.toArray()).from(keySpace, table);
```

Comment:
@pritha-tarento - Any reason for this change? Can this reverted back if the change is not required?

Reply:
This change is required. Scenario is when the required fields passed as parameter this throws exception to not able to pull the records from underlying DB.